integration/kubernetes/confidential/run_tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Copyright (c) 2022 Red Hat
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This is a thin wrapper to call the confidential containers related tests.
+#
+set -e
+
+script_dir=$(dirname "$(readlink -f "$0")")
+cidir="${script_dir}/../../../.ci"
+source "${cidir}/lib.sh"
+
+main() {
```

Comment:
The tests fail because Kubernetes is not initialized at this point.

This `run_tests.sh` script should be almost identical to https://github.com/kata-containers/tests/blob/main/integration/kubernetes/run_kubernetes_tests.sh, except that the list of tests will be different.

At glance I can see two ways of fixing it without duplicating the `run_kubernetes_tests.sh` script code:

 1.  Make `integration/kubernetes/confidential/run_tests.sh` to call `integration/kubernetes/run_kubernetes_tests.sh`. The former script will need to export the `K8S_TEST_UNION` variable in such a way that only the CC bats tests are executed.
 2. Alike the option 1) it will be exporting `K8S_TEST_UNION`properly and calling `integration/kubernetes/run_kubernetes_tests.sh` however from the `cc-kubernetes` Makefile target. I.e. we don't have this `run_tests.sh` script anymore and just call `run_kubernetes_tests.sh` from the makefile target directly.
 
If you have a better solution, please don't hesitate to share. Otherwise I tend to think option 2) would be better.

Reply:
Hi @wainersm we've done what you suggested for option 2 and deleted the `run_tests.sh` script, and then also referenced `run_kubernetes_tests.sh` in the makefile instead! Hopefully this change was what you were looking for 😄